mmdet/models/dense_heads/fcos_head.py

```diff
@@ -80,26 +126,35 @@ def _init_layers(self):
         self.reg_convs = nn.ModuleList()
         for i in range(self.stacked_convs):
             chn = self.in_channels if i == 0 else self.feat_channels
+            if self.dcn_on_last_conv and \
+                    i == self.stacked_convs - 1:
+                conv_cfg = dict(type='DCNv2')
+            else:
+                conv_cfg = self.conv_cfg
+            if self.conv_bias == 'auto':
+                bias = False if self.norm_cfg is None else True
+            else:
+                bias = self.conv_bias
```

Comment:
These four lines can be directly deleted and `self.conv_bias` can be sent to `ConvModule` directly. Please refer to https://github.com/open-mmlab/mmcv/blob/b87e774f66ff638587902bd0db238026d9549259/mmcv/cnn/bricks/conv_module.py#L91